src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/SlhDsa/SlhDsaTestData.cs

```diff
@@ -32,6 +29,396 @@ public static class SlhDsaTestData
             SlhDsaAlgorithm.SlhDsaShake256f,
         ];
 
+        internal static byte[] IetfSlhDsaSha2_128sPublicKeyPkcs8 => field ??= Convert.FromBase64String(@"
+            MDAwCwYJYIZIAWUDBAMUAyEAK4EJ7Hd8qk4fAkzPz5SX2ZGAUJKA9CVq8rB6+AKJ
+            tJQ=");
+
+        internal static byte[] IetfSlhDsaSha2_128sPublicKeyValue => field ??= (
+            "2B8109EC777CAA4E1F024CCFCF9497D9" +
+            "9180509280F4256AF2B07AF80289B494").HexToByteArray();
+
+        internal static byte[] IetfSlhDsaSha2_128sPrivateKeyPkcs8 => field ??= Convert.FromBase64String(@"
+            MFICAQAwCwYJYIZIAWUDBAMUBECiJjvKRYYINlIxYASVI9YhZ3+tkNUetgZ6Mn4N
+            HmSlASuBCex3fKpOHwJMz8+Ul9mRgFCSgPQlavKwevgCibSU");
+
+        internal static byte[] IetfSlhDsaSha2_128sPrivateKeyValue => field ??= (
+            "A2263BCA45860836523160049523D621" +
+            "677FAD90D51EB6067A327E0D1E64A501" +
+            "2B8109EC777CAA4E1F024CCFCF9497D9" +
+            "9180509280F4256AF2B07AF80289B494").HexToByteArray();
+
+        // Generated using openssl CLI.
+        //
+        // Generate private key pem:
+        // > openssl genpkey -algorithm "SLH-DSA-SHA2-128f" > private.pem
+        //
+        // Get secret key:
+        // > openssl asn1parse -in private.pem | tail -1 | cut -d':' -f4
+        //
+        // Get base64 private key info:
+        // > openssl pkey -outform DER -in private.pem | base64 -w64
+        //
+        // Get base64 public key info:
+        // > openssl pkey -outform DER -pubout -in private.pem | base64 -w64
+        //
+        // Get base64 encrypted private key info:
+        // > openssl pkcs8 -topk8 -outform DER -v2 "aes-192-cbc" -v2prf hmacWithSHA384 -iter 1000000 -in private.pem | base64 -w 64
```

Comment:
1,000,000 is a lot for test data. Are we actually trying to open these encrypted PrivateKeyInfos with a million rounds? That will have an adverse affect on CI performance.